spec/lib/bamboozled/employee_spec.rb

```diff
@@ -117,6 +117,22 @@
     end
   end
 
+  describe "#update" do
+    it 'updates an employee in BambooHR' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.